www/front_src/src/helpers/extras/global.js

```diff
@@ -1,91 +1,107 @@
+/* eslint-disable no-multi-assign */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-continue */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-prototype-builtins */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable consistent-return */
+/* eslint-disable func-names */
+
 /*
  * SystemJS global script loading support
  * Extra for the s.js build only
  * (Included by default in system.js build)
  */
-(function (global) {
+(function(global) {
   const systemJSPrototype = global.System.constructor.prototype;
   const isIE = navigator.userAgent.indexOf('Trident') !== -1;
 
   // safari unpredictably lists some new globals first or second in object order
-  let firstGlobalProp, secondGlobalProp, lastGlobalProp;
-  function getGlobalProp () {
+  let firstGlobalProp;
+  let secondGlobalProp;
+  let lastGlobalProp;
+  function getGlobalProp() {
     let cnt = 0;
     let lastProp;
-    for (let p in global) {
+    for (const p in global) {
       // do not check frames cause it could be removed during import
       if (
-        !global.hasOwnProperty(p)
-        || (!isNaN(p) && p < global.length)
-        || (isIE && global[p] && global[p].parent === window)
+        !global.hasOwnProperty(p) ||
+        (!isNaN(p) && p < global.length) ||
+        (isIE && global[p] && global[p].parent === window)
       )
         continue;
-      if (cnt === 0 && p !== firstGlobalProp || cnt === 1 && p !== secondGlobalProp)
+      if (
+        (cnt === 0 && p !== firstGlobalProp) ||
+        (cnt === 1 && p !== secondGlobalProp)
+      )
         return p;
       cnt++;
       lastProp = p;
     }
-    if (lastProp !== lastGlobalProp)
-      return lastProp;
+    if (lastProp !== lastGlobalProp) return lastProp;
   }
 
-  function noteGlobalProps () {
+  function noteGlobalProps() {
     // alternatively Object.keys(global).pop()
     // but this may be faster (pending benchmarks)
     firstGlobalProp = secondGlobalProp = undefined;
-    for (let p in global) {
+    for (const p in global) {
       // do not check frames cause it could be removed during import
       if (
-        !global.hasOwnProperty(p)
-        || (!isNaN(p) && p < global.length)
-        || (isIE && global[p] && global[p].parent === window)
+        !global.hasOwnProperty(p) ||
+        (!isNaN(p) && p < global.length) ||
+        (isIE && global[p] && global[p].parent === window)
       )
         continue;
-      if (!firstGlobalProp)
-        firstGlobalProp = p;
-      else if (!secondGlobalProp)
-        secondGlobalProp = p;
+      if (!firstGlobalProp) firstGlobalProp = p;
```

Comment:
don't know the linting rule for this, but we may use bracket ?

Reply:
The rule accept this syntax : https://eslint.org/docs/rules/curly#multi-line